src/test/skript/tests/syntaxes/structures/StructAnnotate.sk

```diff
@@ -0,0 +1,20 @@
+
+# The test-only structures below fail to parse if an annotation is/isn't present
+# Since annotations are parse-time, this just needs to parse correctly to pass
+
+test does not have an annotation
+
+@annotation here
+test has an annotation
+
+test does not have an annotation
+
+@annotation here
+@annotation here
+test has an annotation
+
+test does not have an annotation
+test does not have an annotation
+
+test "annotation on structure":
+	@the test is done at parsing time, we just need to pass!
```

Comment:
It seems like we ought to print a warning for a section that just contains an annotation, but that probably isn't possible without deeper integration (e.g. at the Config-parsing level)

Reply:
Yes I agree.

Reply:
To follow up on this, it probably isn't possible right now, but in the future it would be good to detect structures with _only_ meta-syntax in.